README.md

```diff
@@ -124,6 +124,25 @@ In order to get the temperature of a particular node, we can use:
 
 `curl "localhost:9600/_opendistro/_performanceanalyzer/rca?name=AllTemperatureDimensions&local=true"`
 
+## Actions API
+This API provides the last suggested action set via the Decision Maker framework. All the suggested action sets are persisted in the SQL tables. This API provides the action set which was published with the latest timestamp.
+
+
+### SAMPLE REQUEST
+
+GET `_opendistro/_performanceanalyzer/actions`
+
+The sample response from above api
+
+```
+{
+    "LastSuggestedActionSet": "[{\"nodeIps\":\"{1.1.1.1,2.2.2.2}\",\"muted\":false,\"actionName\":\"M1\",\"timestamp\":1602190468123,\"nodeIds\":\"{1,2}\",\"summary\":\"MockSummary\",\"actionable\":false,\"coolOffPeriod\":0},
+                                {\"nodeIps\":\"{1.1.1.1,2.2.2.2}\",\"muted\":false,\"actionName\":\"M2\",\"timestamp\":1602190468123,\"nodeIds\":\"{1,2}\",\"summary\":\"MockSummary\",\"actionable\":false,\"coolOffPeriod\":0}]"
```

Comment:
Also let's add an actual action output in this API doc instead of a mock action. The actual action is more informative and helpful, for e.g. the summary field itself is actually a json containing all the action fields for the queue and cache actions.

Reply:
I have removed this from the README. Will do an end to end testing and update this README as a separate PR.